sympy/parsing/tests/test_sympy_parser.py

```diff
@@ -88,6 +88,22 @@ def test_issue_7663():
     e = '2*(x+1)'
     assert parse_expr(e, evaluate=0) == parse_expr(e, evaluate=False)
 
+def test_issue_10560():
+    inputs = {
+        str(4*-3) : -Integer(3)*Integer(4),
```

Comment:
Since Python evaluates what's in parentheses before applying what is without, `str(4*-3)` -> `str(-12)` -> `-12` so you'll just want to write "4*-3" (and similar below). On the rhs, `-Integer(3)*Integer(4)` will become `-12` for similar reasons so you will want to put the *string* result that you are expecting, e.g. "(-3)*4" (and similar).

If you run the tests locally -- I would use `bin/test -k 10560` -- you will know if the tests are going to pass before committing and running tests on Travis.

Finally, there is no need to open a new commit each time you make changes. Simply commit the changes and push the result as in `git push yourname yourbranch` and github will just update everything.